cmd/amp/cli/cli_test.go

```diff
@@ -119,13 +120,15 @@ func runTestSpec(t *testing.T, test *TestSpec) (err error) {
 		startTime := time.Now().UnixNano() / 1000000
```

Comment:
I think you can simplify your time code. Just use `time.now()` and later check elapsed (eg, `elapsed := time.Since(startTime)`).  I believe the units you're using for the timeout are seconds (needs a comment!), so multiply the timeout by time.Second so you can compare against elapsed.